src/NWCClient.ts

```diff
@@ -181,6 +181,35 @@ export class NWCClient {
     return new NWCClient(options);
   }
 
+  static async getWalletServiceSupportedMethods(
+    options?: ConstructorParameters<typeof NWCClient>[0],
```

Comment:
Actually we only need the relay URL here, maybe I should change this to just be a string parameter

Reply:
EDIT: we need to confirm it was the wallet service who posted the event. So I think it's good to keep it like this